framework/util/show_coverage.pl

```diff
@@ -64,12 +64,14 @@ =head1 DESCRIPTION
     # Check for too many filenames
     pod2usage("$0: Too many files given.\n")  if (@ARGV > 1);
 
+my $expected_test_count = 30;
 my $test_count = 0;
 my $tot_line = 0;
 my $tot_exec = 0;
 my $tot_fail = 0;
 my @fields;
 my $filename = '/tmp/test_d4j/coverage';
+my @lines;
```

Comment:
Why is this predeclared?  Could you add `my` to line 94?
(I would do the same for the `@fields` variable, but I realize you didn't write that.)